src/languages/en.js

```diff
@@ -521,6 +524,18 @@ export default {
         phrase5: 'Money transmission is provided by Expensify Payments LLC (NMLS ID:2017010) pursuant to its',
         phrase6: 'licenses',
     },
+    validateCodeForm: {
+        magicCodeNotReceived: "Didn't receive a magic code?",
+        fillTwoFactorAuthOrRecoveryCode: 'Please enter your two factor code or recovery code',
```

Comment:
Also from the figma mockups ([here](https://www.figma.com/file/jBTULCcei4beQaEWyU8N4i/Passwordless?node-id=1%3A114&t=8aWnhwFF95In5xjQ-0)) this should just be `Please enter your two factor code` since we're trying to limit the number of "code" references we're using

Reply:
We use `recovery code` if they enter the incorrect 2FA code with an error message like so (on oldDot): 

![image](https://user-images.githubusercontent.com/16747822/215637228-913c6a76-724d-4019-91c9-3e9bade70ba4.png)

Reply:
Huh I wonder if Figma was updated recently ish? I the original text here used to be accurate 😅  Hope I'm not crazy 🤞 

And now it looks like the text should actually be `Please enter your authenticator code` to match

Reply:
Requested Spanish translation [here](https://expensify.slack.com/archives/C21FRDWCV/p1675162133371469?thread_ts=1671620113.800089&cid=C21FRDWCV)